--- a/src/qt_gui_cpp_shiboken/CMakeLists.txt
+++ b/src/qt_gui_cpp_shiboken/CMakeLists.txt
@@ -43,7 +43,8 @@
 include(${python_qt_binding_DIR}/shiboken_helper.cmake)
 ament_export_dependencies(pluginlib)
 if(shiboken_helper_FOUND)
-  if(Shiboken_VERSION VERSION_GREATER "1.1.1")
+  message(WARNING "Shiboken would segfault when trying to process qt_gui_cpp (see https://bugreports.qt-project.org/browse/PYSIDE-218). Therefore shiboken bindings are being skipped.")
+  #[[if(Shiboken_VERSION VERSION_GREATER "1.1.1")
     # shiboken 1.1.2 and higher will segfault until https://bugreports.qt-project.org/browse/PYSIDE-218 is fixed
     message(WARNING "Shiboken version ${Shiboken_VERSION} would segfault when trying to process qt_gui_cpp (see https://bugreports.qt-project.org/browse/PYSIDE-218). Therefore shiboken bindings are being skipped.")
   else()
@@ -63,5 +64,5 @@
 
     install(TARGETS qt_gui_cpp_shiboken
       DESTINATION ${PYTHON_INSTALL_DIR}/${PROJECT_NAME})
-  endif()
+  endif()]]
 endif()
